apps/src/lib/kits/maker/ui/SetupStep.story.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import SetupStep from './SetupStep';
+
+export default storybook => {
+  return storybook
+    .storiesOf('SetupStep', module)
+    .addStoryTable([
+      {
+        name: 'Succeeded Step',
+        description: 'Step in success state',
+        story: () => (
+          <SetupStep
+            stepName="This step is successful"
+            stepStatus="SUCCEEDED"
```

Comment:
These status constants are already exported from SetupStep.jsx - let's import and use them here.

https://github.com/code-dot-org/code-dot-org/blob/32a79d6f1187f343782141fe4b99f37f8ed37b0a/apps/src/lib/kits/maker/ui/SetupStep.jsx#L5-L10